portality/static/doaj/js/doaj.facetview.theme.js

```diff
@@ -493,15 +493,35 @@ function humanDate(datestr) {
 //////////////////////////////////////////////////////
 // fixed query widget generation
 /////////////////////////////////////////////////////
+
+var doajenvmap = {
```

Comment:
This seems alright, it's only in the facetview theme rather than the vendor library itself. That's partly the point of having a theme, so you can do custom things with it. You already had the fragment "// Code to configure the widget" below in the FV DOAJ theme.